scripts/linters/python_linter.py

```diff
@@ -115,20 +117,127 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_all_job_listed_in_job_registry_file(self):
+        """This function is used to check that all the one-off and audit jobs
+        are registered in jobs_registry.py file.
+        """
+        def _get_jobs_class_names_in_filepath(filepath, base_class_name):
+            """Returns a list of job class names in the given filepath which has
+            the given base class.
+
+            Args:
+                filepath: str. The filepath of the jobs.
+                base_class_name: str. The name of the base class.
+
+            Returns:
+                list(str). A list of class names which exist in the given
+                filepath delivered from the base class.
+            """
+            class_names = []
+            module_path = filepath.replace('/', '.')[:-3]
```

Comment:
First assert that the filepath ends with .py. Then remove '.py'.length rather than "-3" -- no one knows where -3 comes from.

Reply:
Yeah, I had copied this code from another place in the codebase
![image](https://user-images.githubusercontent.com/16653571/79992676-74c01c80-84d1-11ea-8e37-8451a6868652.png)

This file expected to receive a python file, so I'll remove it directly.